docs/Quantization.md

```diff
@@ -212,3 +212,16 @@ Row-wise quantized SparseLengthsWeightedSum is also supported. Similar to the
 above, we compute scales and offsets per row, to be used with the `Data` input
 for the `RowwiseQuantizedSparseLengthsSumNode`. Scales and Offsets are inputs to
 the node. Output of this node is float, matching the Caffe2 implementation.
+
+### Fused Row-wise Quantization
+
+For some backends it may be beneficial to keep each row's scales and offsets
+fused inline with the data. Caffe2 implements nodes with fused storage, such as
+[SparseLengthsWeightedSum](https://caffe2.ai/docs/operators-catalogue.html#sparselengthsweightedsumfused8bitrowwise). Glow
```

Comment:
> uint8 tensor obtained with operator FloatToFused8BitRowwiseQuantized

How do we use the fact that it's uint8 stored, but not int8?

It seems like in C2 loaders we assume it's int8 type
> RETURN_ERR_IF_NOT(dataC->getElementType() == ElemKind::Int8QTy, "Data must be Int8QTy.");